app/src/i18n.ts

```diff
@@ -0,0 +1,18 @@
+import i18n from "i18next"
+import LanguageDetector from "i18next-browser-languagedetector"
+import { initReactI18next } from "react-i18next"
+import * as resources from "./locales"
+
+i18n
+  .use(LanguageDetector)
```

Comment:
[This](https://github.com/i18next/i18next-browser-languageDetector) will fallback on the browser's language, but I imagine we'll want to expose a simple language picker – maybe just a list of languages at the bottom of the homepage or something?

Reply:
I was thinking a list of languages at the bottom would be good! 

Maybe "hello" in each language or "fish" lol?